### DISCUSSION TITLE
What are we waiting for to change?

### DISCUSSION THREAD

**Discussion by natharch** [microCOVID/microCOVID#1185] [Q&A]
Thanks so much for all your hard work on your amazing website. My community house has been using it for many months now, and I recommend it to everyone I can.

Since you've displayed such clear expertise around COVID and you seem to follow all the latest science, I'm curious if you'd be open to offering me your perspective on a question I've been pondering:

If we assume that some percentage of Americans will _never_ agree to be vaccinated, **what will actually change in the future to allow us to be less careful around COVID** (specifically in the United States)?

Is it that the virus will mutate and become less dangerous? Or that we vaccinated people will all develop sufficient immunity after multiple boosters that masks won't be necessary? Or that we’ll all most likely get COVID at some point but we’ll develop powerful treatments so it won’t be a big deal?

I ask because I've had friends say that they've decided to be less careful since they're most likely going to get COVID at some point, so might as well get it over with and get to experience the positive mental health benefits of less anxiety, more time with friends, etc. I struggle to know how to reply since I can't quite articulate how it would be any worse to get COVID now versus a year from now, or five years from now. 

What do you think? What exactly are we waiting for to change?

**Comment by aniola** [comment]
I am not associated with microcovid.org, I am also just an enthusiastic user.

My answer to your question is that pandemics seem to peak and then die down.  If you look at the 1918 flu, it never went away.  But it stopped being pandemic and started being endemic.  I may be getting the details wrong but that's my general sense of it.  

Likewise with the bubonic plague, there would be flareups (typically for a year or two) every decade for _centuries_.  Occasionally the flareups would last as long as a decade, but they never lasted forever.  I know this isn't the bubonic plague but I'm figuring a pandemic is a pandemic.

I think one of the metrics some people (many?) are using is how full the hospitals are: are your local hospitals above capacity?  Still pandemic.  Below capacity?  Frolick while you can.

**Reply by aniola** [reply]
Except apparently I'm wrong, and this particular virus may or may not take that trajectory.
https://www.smithsonianmag.com/science-nature/how-viruses-evolve-180975343/
